test/karma/tail.js

```diff
@@ -0,0 +1,15 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+// Stop the default init functions running to avoid conflicts in tests
```

Comment:
Isn't this really an indicator that we want to be using dependency injection somewhere that we're not?  I don't think it's necessarily required to fix that before landing this, but if you agree, it might be worth adding an XXX comment.